# OpenStax Chemistry, 2nd Ed. &ndash; Dean's Errata Submissions
## Chapter 5
### Chapter 5 &ndash; Examples

**Execute this cell to display chemical formulas.**
It contains a LaTeX require for mhchem, which even applies to later cells.
$$ \require{mhchem} $$

Some applicable formulas.
$$ q_{substance\ a} + q_{substance\ b} = 0 $$
$$ \text{specific heat}\  c = \frac{q}{m\Delta T} $$
$$ \Delta T = T_{final} - T_{initial} = T_f - T_i $$
$$
\begin{align}
q &= c \cdot m \cdot \Delta T \\ 
&= c \cdot m \cdot (T_{final} - T_{initial})
\end{align}
$$
$$ (c_a \cdot m_a \cdot \Delta T_a) + (c_b \cdot m_b \cdot \Delta T_b) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) + (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) = 0 $$
$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$

#### Example 5.3 Solution

Define $a$ to be the metal, and $b$ to be the water.
$$ T_{a\ final} - T_{a\ initial} = - \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) $$
$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$
$$ m_b = 425 g $$
$$ c_b = 4.184 \frac{J}{g {\degree}C} $$
$$ T_{b\ final} = 42.7 {\degree}C $$
$$ T_{b\ initial} = 24.0 {\degree}C $$
$$ m_a = 360.0 g $$
$$ c_a = 0.449 \frac{J}{g {\degree}C} $$
$$ T_{a\ final} = T_{b\ final} = 42.7 {\degree}C $$
$$ T_{a\ initial} = \left(\frac{425 g \cdot 4.184 \frac{J}{g {\degree}C} \cdot (42.7 {\degree}C - 24.0 {\degree}C)}{360.0 g \cdot 0.449 \frac{J}{g {\degree}C}}\right) + 42.7 {\degree}C $$

In [1]:
((425 * 4.184 * (42.7 - 24.0)) / (360.0 * 0.449)) + 42.7

248.41851026973524

$$ T_{a\ initial} = 248 {\degree}C $$

#### Example 5.3 Check Your Learning 1

$$ T_{a\ initial} = \left(\frac{c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})}{c_a \cdot m_a}\right) + T_{a\ final} $$

In [7]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248))

295.6582320904901

In [6]:
((4.184 * 390 * (39.9 - 22.6)) / (0.385 * 248)) + 39.9

335.5582320904901

336 degrees C

#### Example 5.3 Check Your Learning 2

$$ (c_a \cdot m_a \cdot (T_{a\ final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{b\ final} - T_{b\ initial})) $$
Define $a$ to be the metal, and $b$ to be the water.
Note that $T_{a\ final}$ should always equal $T_{b\ final}$.
$$ (c_a \cdot m_a \cdot (T_{final} - T_{a\ initial})) = - (c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})) $$
$$ \require{cancel} $$
$$ \frac{\cancel{c_a \cdot m_a \cdot} (T_{final} - T_{a\ initial})}{\cancel{c_a \cdot m_a}(c_b \cdot m_b)} = - \frac{\cancel{c_b \cdot m_b \cdot} (T_{final} - T_{b\ initial})}{(c_a \cdot m_a)\cancel{c_b \cdot m_b}} $$
$$ \frac{T_{final} - T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final} - T_{b\ initial}}{c_a \cdot m_a} $$
$$ \frac{T_{final}}{c_b \cdot m_b} - \frac{T_{a\ initial}}{c_b \cdot m_b} = - \frac{T_{final}}{c_a \cdot m_a} - \frac{T_{b\ initial}}{c_a \cdot m_a}$$

**Wait, back up**
$$
\begin{align}
c_a \cdot m_a \cdot (T_{final} - T_{a\ initial}) &=
- c_b \cdot m_b \cdot (T_{final} - T_{b\ initial})
\\[1 ex]
% c_a \cdot m_a \cdot T_{final} -
% c_a \cdot m_a \cdot T_{a\ initial} &=
% - c_b \cdot m_b \cdot T_{final} +
% c_b \cdot m_b \cdot T_{b\ initial}
% \\[1 ex]
c_a \cdot m_a \cdot T_{final} -
c_a \cdot m_a \cdot T_{a\ initial} &=
c_b \cdot m_b \cdot T_{b\ initial} -
c_b \cdot m_b \cdot T_{final}
\\[1 ex]
c_a \cdot m_a \cdot T_{final} +
c_b \cdot m_b \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
(c_a \cdot m_a + c_b \cdot m_b) \cdot T_{final} &=
c_b \cdot m_b \cdot T_{b\ initial} +
c_a \cdot m_a \cdot T_{a\ initial}
\\[1 ex]
T_{final} &=
\frac{c_b \cdot m_b \cdot T_{b\ initial} + c_a \cdot m_a \cdot T_{a\ initial}}
     {c_a \cdot m_a + c_b \cdot m_b}
\end{align}
$$

Defining $a$ to be the copper and $b$ to be the water, we have...
$$
T_{final} =
\frac{4.184 J/g {\degree}C \cdot 390 g \cdot 22.6 {\degree}C +
      0.385 J/g {\degree}C \cdot 248 g \cdot 314 {\degree}C}
     {0.385 J/g {\degree}C \cdot 248 g + 4.184 J/g {\degree}C \cdot 390 g}
$$

In [10]:
((4.184 * 390 * 22.6) + (0.385 * 248 * 314)) / ((0.385 * 248) + (4.184 * 390))

38.70828373590237

38.7 degrees C

#### Example 5.4 Solution

$$ \begin{align}
\Delta T = T_{final} - T_{initial} &= T_f - T_i
\\[1ex]
c_{metal} \times m_{metal} \times \Delta T_{metal} &= -
c_{water} \times m_{water} \times \Delta T_{water}
\\[1 ex]
c_{metal} &= - \frac{c_{water} \times m_{water} \times \Delta T_{water}}
                    {m_{metal} \times \Delta T_{metal}}
\\[1 ex]
c_{metal} &= - \frac{4.184 J/g \degree C \times 60.0 g \times (28.5 - 22.0 \degree C)}
                    {59.7 g \times (28.5 - 100.0 \degree C)}
\end{align} $$

In [13]:
- (4.184 * 60.0 * (28.5 - 22.0)) / (59.7 * (28.5 - 100.0))

0.3822750114207401

Specific heat $c_{metal} = 0.382 J / g \degree C$, so the metal is probably Copper.

<p style="color: red">The book said 0.38.  Did the (28.5 - 22.0) term being 2 digits do that?</p>

**<p style="color:red">Check rounding rules.</p>**

#### Example 5.4 Check Your Learning

In [14]:
- (4.184 * 75.0 * (29.7 - 24.0)) / (92.9 * (29.7 - 178.0))

0.12982876620355413

Specific heat $c_{metal} = 0.130 J / g \degree C$ for the silvery-gray metal.

The metal is probably lead, because the only other metal listed in Table 5.1 is gold, which does not have a silvery gray color.

<p style="color: red">The book said 0.13 (and lead).</p>
<p style="color: red"><strong>Check rounding rules.</strong></p>

### Chapter 5 &ndash; Exercises